src/main/java/peppa/Storage.java

```diff
@@ -0,0 +1,71 @@
+package peppa;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+/**
+ * Represents a file for storing tasks and handles all file I/O-related logic.

```

Comment:
Optional: If you would like to, you could add @author as part of the JavaDocs in every class so as to claim "copyrights". 

Note: Ps this is not part of the Java Coding Standards but I just feel since you have made this pretty unique and put in a lot of hardwork, might be good to call dips on "copyright".